pkg/authorization/api/v1/conversion.go

```diff
@@ -130,7 +131,10 @@ func convert_api_ResourceAccessReviewResponse_To_v1_ResourceAccessReviewResponse
 	return nil
 }
 
-func convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
+func Convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
+	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
```

Comment:
Just because we haven't done that upstream yet (which I have an issue
opened for).

On Tue, Mar 8, 2016 at 6:51 AM, David Eads <notifications@github.com> wrote:

> In pkg/authorization/api/v1/conversion.go
> <https://github.com/openshift/origin/pull/7814#discussion_r55354436>:
>
> > @@ -130,7 +131,10 @@ func convert_api_ResourceAccessReviewResponse_To_v1_ResourceAccessReviewResponse
> >  	return nil
> >  }
> >
> > -func convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
> > +func Convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
> > +	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
>
> If you're worried about reflection cost, why not go ahead and make a named
> function for it and make the call directly? Even registering and calling an
> empty function would do a pretty good job of making sure the code doesn't
> slip.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7814/files#r55354436>.
>